style.css

```diff
@@ -383,3 +393,124 @@ header {
   background: #6070ff;
   color: #fff;
 }
+
+.nav-desktop {
+  display: none;
+}
+
+@media all and (min-width: 768px) {
+  /* ===============================*\
+           Header
+*\=============================== */
+
+  .nav-desktop {
+    display: block;
+  }
+
+  .menu-desktop {
+    list-style: none;
+    display: flex;
+    gap: 20px;
+  }
+
+  header {
+    padding: 0;
+  }
+
+  header .navbar .menu-desktop a {
+    font-size: 15px;
+    line-height: 20px;
+    font-style: normal;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  header .navbar .menu-desktop a:hover {
+    color: #6070ff;
+  }
+
+  .hamb-bu {
+    display: none;
+  }
+
+  .navbar {
+    padding: 15px 142px;
+  }
+
+  /* ===============================*\
+           headline
+*\=============================== */
+
+  #headline {
+    background-image: url("images/Header-Desktop.svg");
+    justify-content: center;
+    padding: 0 110px;
+  }
+
+  #headline h1 {
+    font-size: 48px;
+  }
+
+  #headline p {
+    font-size: 20px;
+  }
+
+  /* ===============================*\
+           Work
+*\=============================== */
+  .portfolio-snapshot {
+    flex-direction: row;
+  }
+
+  #work {
+    padding: 142px;
+    margin: 0;
+  }
+
+  .img {
+    width: 100%;
+    cursor: pointer;
+  }
+
+  .order {
+    order: 2;
+  }
+
+  /* ===============================*\
+           About me
+*\=============================== */
+
+  #about-section {
+    padding: 114px 142px;
+    display: flex;
+    gap: 30px;
+  }
+
+  .about {
+    width: 50%;
+    padding: 0;
+  }
+
+  .language {
+    flex-direction: row;
+    width: 50%;
+  }
```

Comment:
I think you did a great job here however there is something to improve. The relation between these two containers is 40% to 60%. Kindly change the width of these containers to make your project stick to the Figma template.  :pray: :pray: 
Your design
![image](https://user-images.githubusercontent.com/98363075/200099468-33b1311b-6bc5-4ee7-a812-74f30a910ee8.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/200099483-84c9d769-63df-4449-a248-b775fc470723.png)